scripts/import-cp-contexts.sh

```diff
@@ -0,0 +1,253 @@
+#!/usr/bin/env bash
+# Copyright 2024 The KubeStellar Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+set -e
+
+
+arg_names=""
+arg_kubeconfig=""
+arg_context=""
+arg_host=""
+arg_merge=false
+arg_output=""
+arg_verbose=true
+
+
+display_help() {
+  cat << EOF
+Usage: $0 [--kubeconfig <filename>] [--context <name>] [--names <list of names>] [--replace-localhost <address>] [--merge] [-o <filename>] [-V] [-X]
```

Comment:
It's a little awkward that `--help` does not tell the reader about both forms of each flag.

Not a blocker.